dev/ci/README.md

```diff
@@ -178,3 +178,14 @@ but if you wish to save more time you can skip the job by setting
 This means you will need to change its value when the Docker image
 needs to be updated. You can do so for a single pipeline by starting
 it through the web interface.
+
+### Artifacts
+
+Build artifacts from Gitlab can be linked / downloaded in a systematic
+way, see [Gitlab's documentation](https://docs.gitlab.com/ce/user/project/pipelines/job_artifacts.html#downloading-the-latest-job-artifacts)
+for more information. For example, to access the documentation of a particular branch, you can do:
+
+https://gitlab.com/coq/coq/-/jobs/artifacts/$branch/file/_install_ci/share/doc/coq/sphinx/html/index.html?job=build:base
+
+Browsing artifacts is also possible:
+https://gitlab.com/coq/coq/-/jobs/artifacts/$branch/browse/_install_ci/share/doc/coq/?job=build:base
```

Comment:
I would rather use master in the examples. This link is going to be clickable and "broken"

Reply:
Both comments fixed, thanks.